lms/templates/courseware/progress.html

```diff
@@ -180,12 +180,30 @@ <h4 class="hd hd-4">
                                             %endif
                                         </p>
                                         %if len(section.problem_scores.values()) > 0:
-                                        <dl class="scores">
-                                            <dt class="hd hd-6">${ _("Problem Scores: ") if section.graded else _("Practice Scores: ")}</dt>
-                                            %for score in section.problem_scores.values():
-                                            <dd>${"{0:.3n}/{1:.3n}".format(float(score.earned),float(score.possible))}</dd>
-                                            %endfor
-                                        </dl>
+                                          %if section.show_grades(staff_access):
+                                          <dl class="scores">
```

Comment:
Is there a use case for granular staff/non-staff access users? Why not use the template global 'staff_access'? 

`section.show_grades(staff_access)` seems to be defaulting to True for me on devstack -- I believe the variable exists regardless of truthiness, but it's possible to be less granular per module in this instance (I think -- push back if I'm wrong, and I often am).